ydb/core/kqp/proxy_service/kqp_proxy_service_impl.h

```diff
@@ -295,7 +308,7 @@ class TLocalSessionsRegistry {
         return LocalSessions.size();
     }
 
-    const TKqpSessionInfo* FindPtr(const TString& sessionId) const {
```

Comment:
Lets make a method to set QueryText instead of removing const. It is unsafe to allow to change arbitrary fields for already created session. See AttachSession for example